tests/coap/README.md

```diff
@@ -21,6 +21,12 @@ This application is a work in progress demo of running CoAP with OSCORE/EDHOC se
   up to the maximum number of security contexts that are stored (currently 4).
 * There is also `./fauxhoc.py`, which did EDHOC manually before it was integrated in aiocoap.
 
+### Variation
+
+* CoAP in NoSec mode: Building a smaller binary at the cost of confidentiality and integrity protection.
+    * In `laze.yml`, replace `coap-server-config-demokeys` with `coap-server-config-unprotected`.
```

Comment:
Is this the right way to do a demo, or can the example do something softer than "selects", and the user can push things over to another one (deselcting the demokeys by virtue of the conflict)?

Reply:
Obsoleted by the `provides_unique` construction: now it can be overridden on the command line, and the README is updated to reflect that.